public/docs/ts/latest/guide/server-communication.jade

```diff
@@ -628,6 +629,26 @@ block wikipedia-jsonp+
       You added the `debounceTime`, `distinctUntilChanged`, and `switchMap` operators to the RxJS `Observable` class
       in `rxjs-operators` as [described above](#rxjs).
 
+a#xsrf
+.l-main-section
+:marked
+  ## Guarding against Cross-Site Request Forgery
+
+  In a cross-site request forgery (CSRF or XSRF), an attacker tricks the user into visiting
```

Comment:
why do we repeat the same content here? should this link to the original doc, so we don't have two sources of truth that can diverge?